corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -151,59 +151,95 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
-          <thead>
-          <th></th>
-          <th>{% trans "Case Property" %}</th>
-          <th>{% trans "Data Type" %}</th>
-          <th>{% trans "Description" %}</th>
+        <button data-bind="click: $root.showDeprecated, visible: !showAll()" class="btn btn-default pull-right">{% trans "Show Deprecated" %}</button>
+        <button data-bind="click: $root.hideDeprecated, visible: showAll" class="btn btn-default pull-right">{% trans "Hide Deprecated" %}</button>
+        <div id="data-dictionary-table">
+          <div class="table-row table-header">
+          <div class="row-item-small"></div>
+          <div class="row-item">{% trans "Case Property" %}</div>
+          <div class="row-item">{% trans "Data Type" %}</div>
+          <div class="row-item">{% trans "Description" %}</div>
           {% if fhir_integration_enabled %}
-            <th>{% trans "FHIR Resource Property Path" %}</th>
+            <div class="row-item">{% trans "FHIR Resource Property Path" %}</div>
           {% endif %}
-          <th>{% include "data_dictionary/partials/valid_values_th_content.html" %}</th>
+          <div class="row-item">{% include "data_dictionary/partials/valid_values_th_content.html" %}</div>
           {% if not request.is_view_only %}
-            <th></th>
+            <div class="row-item-small"></div>
           {% endif %}
-          </thead>
-          <tbody data-bind="sortable: casePropertyList">
-          <!-- ko if: isGroup -->
-          <tr class="active">
-            <td><i class="fa" data-bind="css: { 'fa-plus-square': !expanded(), 'fa-minus-square': expanded() }, click: $root.toggleGroup"></i></td>
-            <td data-bind="text: name ? name : 'No Group'"></td>
-            <td>{% trans "Case Property Group" %}</td>
-            <td></td>
-            <td></td>
-            {% if not request.is_view_only %}
-              <td></td>
-            {% endif %}
+          </div>
+          <div data-bind="sortable: { data: caseGroupList, connectClass: 'groups', options: { handle: 'i.sortable-handle' } }">
+          <div>
+          <!-- ko if: deprecated() && !isDeprecatedOnLoad && properties().length > 0 -->
+          <div class="group-deprecated" style="padding: 8px 4px;">
+            <b data-bind="text: name()"></b>
+            {% trans "group's properties will be moved to" %}
+            <b>"No Group"</b>
+          </div>
+          <!-- /ko -->
+          <div class="table-row group" data-bind="css: { 'group-deprecated': deprecated() }, visible: !isDeprecatedOnLoad || $root.showAll()">
+            <div class="row-item-small">
+              {% if not request.is_view_only %}
+              <i class="sortable-handle fa fa-arrows-v"></i>
+              {% endif %}
+              <i class="fa"
+                data-bind="css: { 'fa-plus-square': !expanded(), 'fa-minus-square': expanded() }, click: toggleExpanded"
+                style="margin-left: 8px;"></i>
```

Comment:
can we move this to a css file?

Reply:
I was using inline styles for adding margin and padding to some other elements as well. I have refactored those to be in the format of [Bootstrap 5 spacing util](https://getbootstrap.com/docs/5.0/utilities/spacing/) classes, that will make it easier to migrate to them in future. [74a6122](https://github.com/dimagi/commcare-hq/pull/32878/commits/74a612288601c0f3009995067be1dc044bc8aed2)